news/2018-06-22-cwc-2018-recap-round-of-16.md

```diff
@@ -0,0 +1,59 @@
+---
+layout: post
+title: "osu!catch World Cup 2018: Round of 16 Recap"
+date: 2018-06-22 12:00:00 +0000
+---
+
+Two weeks of frantic fruit-catching in the osu!catch World Cup 2018 completed, and several more still to look forward to - but who triumphed at the top of the tree, and who fell off the platter of progress in the Round of 16? Read on to find out more!
+
+![](/wiki/shared/news/banners/CWC_2018_banner.jpg)
+
+Hello once again one and all, [JBHyperion](https://osu.ppy.sh/users/4879508) here to deliver your weekly round-up on all the action from this year's edition of the osu!catch World Cup. Following on from one of the most incredible Group Stages ever seen in an osu! World Cup, last weekend's Round of 16 saw our remaining teams each jostling for position in their march to becoming this year's *osu!catch Champions*.
+
+A further eight matches were shown live on stream this past Sunday, each delivering further energy and excitement, shocks and swerves, as the tournament bracket continues to take shape. 
+
+Wanting to inject a little more flair and personality into things, the first match was preceded by an osu! Coffee Hour “pre-show” where fellow commentator [Dohland](https://osu.ppy.sh/u/5220511) and myself ran down a recap of the best moments from the previous week's Group Stages, as well as most anticipated moments and predictions for each of the Round of 16 matches.
+
+If you missed any of the action and analysis, or are just eager to relive it a second time, VODs for all eight matches and pre-show are available on the [osu!live Twitch channel](https://www.twitch.tv/osulive).
+
+<iframe src="https://player.twitch.tv/?autoplay=false&video=v274192014" frameborder="0" allowfullscreen="true" scrolling="no" height="378" width="100%"></iframe>\
```

Comment:
is this appended backstroke intended ?

Reply:
Using it to force a new line, my previous github markdown of </br> to do this didn't seem to work as intended